api/tests/Api/ContentNodes/ColumnLayout/CreateColumnLayoutTest.php

```diff
@@ -12,7 +12,7 @@ class CreateColumnLayoutTest extends CreateContentNodeTestCase {
     public function setUp(): void {
         parent::setUp();
 
-        $this->endpoint = '/content_node/column_layouts';
+        $this->endpoint = '/column_layouts';
```

Comment:
I needed to separate `routePrefix` (=`"/content_node"`) and `uriTemplate` (=`"/column_layouts"`) in order to compile the correct operationName in `getExamplePayload`:
https://github.com/ecamp/ecamp3/pull/3176/files#diff-1c80e80e582855ae7b4bbbdfce8894f47dcb3ef2a8e5093dfef03cba5cebf9c7R148

This was needed, because currently in api-platform `routePrefix` is not part of the operation name. I earlier thought this was by purpose, but later realized this is a bug. It's registered upstream and a PR already exists. After this is merged/released, I can revert the change.

https://github.com/api-platform/core/issues/5203
https://github.com/api-platform/core/pull/5208

Reply:
Upgraded to api-platform 3.0.7 and fixed with https://github.com/ecamp/ecamp3/pull/3176/commits/83b49cf9435bae099aab1c57dbc625ed0d2225df